src/redis-cli.c

```diff
@@ -8857,6 +8870,8 @@ int main(int argc, char **argv) {
     memset(&config.sslconfig, 0, sizeof(config.sslconfig));
     config.conn_info.hostip = sdsnew("127.0.0.1");
     config.conn_info.hostport = 6379;
+    config.connect_timeout.tv_sec = 60;
+    config.connect_timeout.tv_usec = 0;
```

Comment:
I wonder what was the default connection timeout before this change? Perhaps 2 hours (Linux default)? If so then maybe it's better to keep the previous default for backward compatibility?

Reply:
I think it's a bit long, but good point. i do not konw the default value :)

Reply:
Actually I couldn't find anything specific about a default connection timeout. So I'm ok with leaving it at 60 sec.

Reply:
I think 60 seconds is fine too, but just for reference: In hiredis-cluster, there is code like this:

```C
    if (cc->connect_timeout) {
        c = redisConnectWithTimeout(node->host, node->port,
                                    *cc->connect_timeout);
    } else {
        c = redisConnect(node->host, node->port);
    }
```